docs/development/select-screen.md

```diff
@@ -120,10 +120,10 @@ The select screen will paginate the choices based on the `rows` and `cols` param
 }
 ```
 
-The page displayed when first loaded can be changed by setting `choice` in transient data, under the key for the screen, e.g:
+The page displayed when first loaded can be changed by setting a unique key on the element in the collection, e.g:
```

Comment:
Doesn't the value need to be 'selected' for the select screen to display it?

Reply:
not value, key, e.g: 

```{ id: "1", key: "selected", value: true }```

Reply:
yes... I'm having a moment lol